sdk/state/integrationtests/state_test.go

```diff
@@ -42,37 +42,69 @@ func TestOpenUpdatesUncoordinatedClose(t *testing.T) {
 
 	// Open
 	t.Log("Open...")
+	// I signs txClose
 	open, err := initiatorChannel.ProposeOpen(state.OpenParams{Asset: asset, AssetLimit: assetLimit})
 	require.NoError(t, err)
-	for {
+	assert.ElementsMatch(t,
+		[]int{1, 0, 0},
+		[]int{len(open.CloseSignatures), len(open.DeclarationSignatures), len(open.FormationSignatures)},
+	)
```

Comment:
❗ `ElementsMatch` checks that the elements of the array are the same, but doesn't require them to be in the same order which means this assertion will still pass even if `len(open.CloseSignatures)` is zero and one of the others have a value of one. I believe you need to use `Equal`. Or, to improve the readability of this, simply write three simple assertions.
```suggestion
	assert.Len(t, open.CloseSignatures, 1)
	assert.Len(t, open.DeclarationSignatures, 0)
	assert.Len(t, open.FormationSignatures, 0)
```

Reply:
oops thanks, changed
https://github.com/stellar/experimental-payment-channels/pull/125/commits/286d5b91c92ef1af0e40a266d144667660c09134